ansible/roles/ec2_provision/tasks/main.yml

```diff
@@ -152,6 +152,9 @@
       user: "{{ ec2_user }}"
       Name: "{{ ec2_user }}-origin3-dev"
     wait: yes
+    volumes:
+    - device_name: /dev/sda1
```

Comment:
This looks strange. But it needs to match what the AMI specifies and RHEL AMI's specify this device.
See the bottom of https://github.com/ansible/ansible/issues/20347. I confirmed it looks as they stated on the AMI we're using.